data/language/dutch.txt

```diff
@@ -1709,7 +1709,7 @@ STR_1707    :Teveel werknemers in dit park
 STR_1708    :{SMALLFONT}{BLACK}Werkgebied van deze werknemer instellen
 STR_1709    :Werknemer ontslaan
 STR_1710    :Ja
-STR_1711    :{WINDOW_COLOUR_1}Weet je zeker dat je {STRINGID} wilt ontslaan?
+STR_1711    :{WINDOW_COLOUR_1}Weet je zeker dat je {STRINGID} wil ontslaan?
```

Comment:
In de meeste geschreven tekst wordt nog gewoon 'wilt' gebruikt en 'wil' is eerder uitzondering dan regel op plekken anders dan fora en berichtservices. Ik geef dus toch de voorkeur aan de eerste.